src/index.html

```diff
@@ -9,10 +9,135 @@
     <title>Calendar</title>
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="./styles/main.scss"
     />
```

Comment:
You should not link .scss files directly in HTML. Instead, you should compile your .scss files to .css and then link the .css file in your HTML.